### DISCUSSION TITLE
Admin documentation

### DISCUSSION THREAD

**Discussion by benhartwich** [microsoft/dotnet-podcasts#19] [Q&A]
Hi,

I wonder, if there is any admin documentation on how to bring new podcasts to this infrastructure or delete existing podcasts? I haven´t found any "admin login" at the deployed software.

**Comment by jamesmontemagno** [comment]
There isn't really any admin mode at all for the podcasts.

For the keynote demo we have an "Ingestion" api that can be enabled that gives you an endpoint to submit a new podcast to https://github.com/microsoft/dotnet-podcasts/tree/main/docs/demos/azurecontainerapps

The only way to remove podcasts is to connect to the database and remove manually. Wasn't in the scope of the project, but a simple Blazor app could totally be spun up.
